sdk/storage/arm-storage/CHANGELOG.md

```diff
@@ -1,4 +1,51 @@
 # Release History
+
+## 19.0.0 (2025-08-15)
+
+### Features Added
+  - Added Interface DualStackEndpointPreference
+  - Added Interface EncryptionInTransit
+  - Added Interface NfsSetting
+  - Added Interface Placement
+  - Added Interface SkuInformationLocationInfoItem
+  - Added Interface SmbOAuthSettings
+  - Added Interface StorageAccountIpv6Endpoints
+  - Interface AzureFilesIdentityBasedAuthentication has a new optional parameter smbOAuthSettings
+  - Interface Endpoints has a new optional parameter ipv6Endpoints
+  - Interface NetworkRuleSet has a new optional parameter ipv6Rules
+  - Interface ProtocolSettings has a new optional parameter nfs
+  - Interface SkuInformation has a new optional parameter locationInfo
+  - Interface SmbSetting has a new optional parameter encryptionInTransit
+  - Interface StorageAccount has a new optional parameter dualStackEndpointPreference
+  - Interface StorageAccount has a new optional parameter placement
+  - Interface StorageAccount has a new optional parameter zones
+  - Interface StorageAccountCreateParameters has a new optional parameter dualStackEndpointPreference
+  - Interface StorageAccountCreateParameters has a new optional parameter placement
+  - Interface StorageAccountCreateParameters has a new optional parameter zones
+  - Interface StorageAccountUpdateParameters has a new optional parameter dualStackEndpointPreference
+  - Interface StorageAccountUpdateParameters has a new optional parameter placement
+  - Interface StorageAccountUpdateParameters has a new optional parameter zones
+  - Interface StorageTaskAssignmentsCreateHeaders has a new optional parameter azureAsyncOperation
+  - Interface StorageTaskAssignmentsDeleteHeaders has a new optional parameter azureAsyncOperation
+  - Interface StorageTaskAssignmentsUpdateHeaders has a new optional parameter azureAsyncOperation
+  - Added Type Alias ZonePlacementPolicy
+  - Added Enum KnownTriggerType
+  - Added Enum KnownZonePlacementPolicy
+
+### Breaking Changes
+  - Operation StorageAccounts.beginCreate has a new signature
+  - Operation StorageAccounts.beginCreateAndWait has a new signature
+  - Operation StorageAccounts.getProperties has a new signature
+  - Operation StorageAccounts.update has a new signature
+  - Operation StorageTaskAssignments.beginCreate has a new signature
+  - Operation StorageTaskAssignments.beginCreateAndWait has a new signature
+  - Operation StorageTaskAssignments.beginUpdate has a new signature
+  - Operation StorageTaskAssignments.beginUpdateAndWait has a new signature
+  - Operation StorageTaskAssignments.get has a new signature
```

Comment:
A lot of new signature breakings but I din't notice there exists api view changes. where is the breaking from?